src/components/Checkbox/Checkbox.jsx

```diff
@@ -8,42 +8,47 @@ import { getDataAttrs, getDataProps } from '../../dataUtils';
 class Checkbox extends React.Component {
   render() {
     const {
-      isBold,
       checked,
+      data,
```

Comment:
would be nice if prettier sorted these kind of destructuring for us automatically

Reply:
maybe we should do it all in one go at some pt. ive also been doing it file by file but it can make pr review noisy